lib/Conversion/FIRRTLToHW/LowerToHW.cpp

```diff
@@ -870,6 +870,16 @@ FIRRTLModuleLowering::lowerExtModule(FExtModuleOp oldModule,
   auto parameters = getHWParameters(oldModule, /*ignoreValues=*/true);
   auto newModule = builder.create<hw::HWModuleExternOp>(
       oldModule.getLoc(), nameAttr, ports, verilogName, parameters);
+  auto hasOutputPort = [&]() {
```

Comment:
bool isOutputport = llvm::is_any_of(firrtlPorts, isOutput)
This seems a gratuitous use of a lambda.